packages/models-library/src/models_library/projects_state.py

```diff
@@ -99,6 +91,23 @@ def check_status_compatible(cls, v, values):
             raise ValueError(msg)
         return v
 
+    @validator("owner", always=True)
+    @classmethod
+    def check_owner_compatible(cls, v, values):
+        if (
+            values["value"] is True
```

Comment:
This was the previous validator:
```   
    @validator("owner", pre=True, always=True)
    @classmethod
    def check_not_null(cls, v, values):
        if values["value"] is True and v is None:
            msg = "value cannot be None when project is locked"
            raise ValueError(msg)
        return v
```
I have extended it with the additional check on status. Yes ordering is important, I guess as value is defined as first so that's the reason why it was always working.

Reply:
You were right, I needed to change the ordering, basically "owner" check already expects that "status" check was done before :+1: